scripts/helper/powervs/parameters.sh

```diff
@@ -163,7 +163,20 @@ function setup_remote_oc_use () {
 		echo "$etc_hosts_entries $append_urls" >> /tmp/hosts.1
 		sudo mv /tmp/hosts.1 /etc/hosts 
 
+		# Preserve variables that are needed after cluster creation.  Mostly, by run_fio.sh
```

Comment:
You pulled this file into your PR by accident.    This is part of the new fio automation